constellation_base/models.py

```diff
@@ -39,3 +42,19 @@ def settings_dict(self):
             'organization': self.organization,
             'titlebar_links': self.titlebar_links,
         }
+
+
+class LocalGroupACLEntry(models.Model):
+    """ACL for Local Groups that don't exist in other authentication sources"""
+    group = models.ForeignKey(Group)
+
+    # This is a string since the user might not exist at the time they log in
+    # and need this to be applied
+    username = models.CharField(max_length=150)
```

Comment:
This does not handle the case where you have updated membership in one group and removed members from another since the user's individual entry still exists in the table.  Thus, if you removed the list for a particular group, they'd still be added to the group on next login.